vulkan_renderer/src/renderer/vulkan/vulkan_swapchain.c

```diff
@@ -283,9 +305,11 @@ static void create(vulkan_context* context, u32 width, u32 height, renderer_conf
             formatted_name,
             image);
 
+        char image_name[32] = "__kohi_default_depth_texture_0_";
+        image_name[30] = '0' + i;
```

Comment:
will fail for more than 9 images. Consider `sprintf(image_name, "__kohi_default_depth_texture_%u_",  (unsigned)i)`